Code/Engine/Core/World/EventMessageHandlerComponent.h

```diff
@@ -6,14 +6,47 @@
 
 struct ezEventMessage;
 
+/// \brief Base class for ezEventMessageHandlerComponent. This can be used if global message handling is not needed and some options shall not show up in the UI.
+class EZ_CORE_DLL ezEventMessageHandlerBaseComponent : public ezComponent
```

Comment:
In our usual naming scheme we put Base at the end of the name, so this should be ezEventMessageHandlerComponent**Base**

Reply:
After the other refactor, we probably don't need this change anymore, anyway.